LightBlue/Standalone/StandaloneMetadataStore.cs

```diff
@@ -21,11 +21,20 @@ public void WriteToStreamAndClose(FileStream fileStream)
 
         public static StandaloneMetadataStore ReadFromStream(FileStream fileStream)
         {
-            using (var reader = new StreamReader(fileStream, System.Text.Encoding.UTF8, true, 1024, true))
+            using (var reader = new StreamReader(fileStream, System.Text.Encoding.UTF8, true, 4096, true))
```

Comment:
Given the significance of matching this to the filestream buffer length I think we should make `StandaloneAzureBlockBlob.BufferSize` internal and reference it here